src/ui/FocusTrap/FocusTrap.ts

```diff
@@ -0,0 +1,112 @@
+import _ = require('underscore');
```

Comment:
Import only the few methods you need for underscore

Reply:
Woops, fixing that.